### PR TITLE
EN-53806 Update OpenDataNetwork.com footer

### PR DESCRIPTION
**Pull request by karrysa**
Replaced "Powered by Socrata" with "Empowered by Tyler Technologies".

### CODE REVIEW

**Comment by karrysa** [comment]
Changed the footer logo and also the one on the side navigation as shown in the screenshots.

Screen Shot 2022-06-24 at 4 13 52 PM
Screen Shot 2022-06-24 at 4 14 49 PM

**Comment by karrysa** [comment]
I showed the changes to Melissa.

**Comment by karrysa** [comment]
When I built locally, package-lock.json was changed. Should I commit this file also?

File: README.md [hunk 1]
```diff
@@ -47,6 +45,9 @@ Run `./flush-memcache.sh` to flush the development cache.
 
 The build process is automated using `gulp` and configured with `gulpfile.js`.
 
+Verify gulp is installed with `gulp --version`. If it's not installed, run
```
Comment (progman32) [inline README.md:48]: Why is the `npm install --save-dev ...` step required? Gulp is already present in `package.json`'s `devDependencies`.

File: README.md [hunk 2]
```diff
@@ -111,3 +112,8 @@ Check LastPass for Heroku access.
 
 The ODN uses the Heroku [MemJS](https://github.com/alevy/memjs) add-on
 for memcached.
+
+#### Troubleshooting
+
+For Tyler Technologies employees, refer to [troubleshooting](https://socrata.atlassian.net/wiki/spaces/ONCALL/pages/2158625000/OpsDoc+-+opendatanetwork.com)
+for further help.
```
Comment (tkbrigham) [inline README.md:119]: 🙇

File: README.md [hunk 3]
```diff
@@ -47,6 +45,9 @@ Run `./flush-memcache.sh` to flush the development cache.
 
 The build process is automated using `gulp` and configured with `gulpfile.js`.
 
+Verify gulp is installed with `gulp --version`. If it's not installed, run
+`npm install --global gulp-cli` and `npm install --save-dev gulp`.
```
[the remark this answers is not in the document]
Comment (karrysa) [reply README.md]: I removed it.

**Comment by karrysa** [comment]
If you click on "Join the Network", I noticed the old logo was in that footer, so I replaced it with the new Tyler one.

Screen Shot 2022-06-27 at 4 09 26 PM

**Review by progman32 — APPROVED** [review]

**Pull request merged**